nike_project/lib/page/SignUp/sign_up_page_body.dart

```diff
@@ -0,0 +1,252 @@
+  bool _submitted = false;
+  bool isPasswordVisible = true;
+  bool isLoading = true;
+
+  ButtonState state = ButtonState.init;
+
+  void _submit() async {
+    setState(() => _submitted = true);
+    if (_key.currentState!.validate()) {
+      setState(() {
+        state = ButtonState.loading;
+      });
+      await Future.delayed(const Duration(seconds: 3));
+      print("email: " + email + "-password: " + password);
+      setState(() {
+        state = ButtonState.done;
+        _submitted = false;
+      });
+      _emailController.clear();
+      _passwordController.clear();
+      await Future.delayed(const Duration(seconds: 3));
+      setState(() {
+        state = ButtonState.init;
+      });
+    }
+  }
+
+  Widget buildSmallButton(bool isDone) {
+    final color = isDone ? Colors.green : Colors.blueAccent.shade400;
+    return Container(
+      decoration: BoxDecoration(shape: BoxShape.circle, color: color),
+      child: Center(
+        child: isDone
+            ? const Icon(
+                Icons.done,
+                size: 52,
+                color: Colors.white,
+              )
+            : const CircularProgressIndicator(
+                backgroundColor: Colors.blueAccent,
+                valueColor: AlwaysStoppedAnimation<Color>(Colors.white),
+              ),
+      ),
+    );
+  }
+
+  ListView buildButtonLogin() {
+    return ListView(
+      scrollDirection: Axis.vertical,
+      shrinkWrap: true,
+      children: <Widget>[
+        ElevatedButtonCustom(
+            width: MediaQuery.of(context).size.width,
+            borderRadius: 10,
+            child: const Text("Login In"),
+            color: Colors.blueAccent.shade400,
+            height: MediaQuery.of(context).size.height * 0.06,
+            onPressed: _submit),
+        const SizedBox(
+          height: 20,
+        ),
+        SocialSignInButton(
+          width: MediaQuery.of(context).size.height * 0.6,
+          text: "Sign In With Google",
+          color: Colors.white,
+          textColor: Colors.black,
+          onPressed: () {},
+          borderRadius: 10.0,
+          height: MediaQuery.of(context).size.height * 0.06,
+          assertName: 'assets/google-48.png',
+        ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    // print(email);
+    final bool isDone = state == ButtonState.done;
+    final bool isStretched = state == ButtonState.init;
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        const SizedBox(
+          height: 10,
+        ),
+        const ButtonBack(),
+        const SizedBox(
+          height: 20,
+        ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: const <Widget>[
+            // Lottie.asset(
+            //   "assets/71361-sign-in.json",
+            //   width: MediaQuery.of(context).size.width * 0.5,
+            // ),
```

Comment:
as the above comment, please remove comment no necessary

Reply:
sory man this comment of me because i use image assest.json device lag then i comment late